### PR TITLE
Wrap kaldi_pybind to kaldi package.

### PR DESCRIPTION
**Pull request by csukuangfj**
Now everything lives in the `kaldi` package and the user
does not need to use `kaldi_pybind` any more.

### CODE REVIEW

**Comment by csukuangfj** [comment]
@danpovey @naxingyu 
please have a review.

File: src/pybind/kaldi/__init__.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import os
+import sys
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.insert(0, os.path.dirname(__file__))
+
+from kaldi_pybind import *
+
```
Comment (danpovey) [inline src/pybind/kaldi/__init__.py]: I suppose there are going to be a lot of decisions about which things to put in the global `kaldi` namespace.  But it's OK.

Comment (csukuangfj) [reply src/pybind/kaldi/__init__.py]: There are lots of classes in `kaldi_pybind` and it is a pain to list each one explicitly.

Comment (danpovey) [reply src/pybind/kaldi/__init__.py]: Sure, that's OK; kaldi is just a thin wrapper for kaldi_pybind.  I was actually talking about the lines below, where you import things like Matrix.  But as I said, it's OK.

**Comment by danpovey** [comment]
anyway this looks fine; merging.

**Pull request merged**